### PR TITLE
Bug 1120926 - Update StatusBar page object to use the minimized and maxi...

### PR DESCRIPTION
**Pull request by bebef1987**
...mized status bars

### CODE REVIEW

**Comment by try-server-hook** [comment]
[bebef1987 Florin Strugariu (bebef1987)](https://github.com/bebef1987) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d9af8d2e20e0)

**Comment by try-server-hook** [comment]
[bebef1987 Florin Strugariu (bebef1987)](https://github.com/bebef1987) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0977f5eccdf4)

File: tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py [hunk 1]
```diff
@@ -8,9 +8,30 @@
...
-    _status_bar_maximized_wrapper = (By.ID, 'statusbar-maximized-wrapper')
-    _status_bar_minimized_wrapper = (By.ID, 'statusbar-minimized-wrapper')
+
+    _status_bar_maximized_locator = (By.ID, 'statusbar-maximized-wrapper')
+    _status_bar_minimized_locator = (By.ID, 'statusbar-minimized-wrapper')
+
+
+    @property
+    def minimized(self):
+        """
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py]: Thanks for this documentation!

File: tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py [hunk 2]
```diff
@@ -8,9 +8,30 @@
...
+        Status bar in it's minimized form. When the RocketBar is at the same level with the connectivity icons.
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py]: Nit: "in its".

File: tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py [hunk 3]
```diff
@@ -8,9 +8,30 @@
...
+        Usually seen when a app is open
+        """
+        mini = self.root_element.find_element(*self._status_bar_minimized_locator)
+        return self.StatusBarRegion(self.marionette, mini)
+
+
+    @property
+    def maximized(self):
+        """
+        Status bar in it's maximized form. When the RocketBar under the connectivity icons.
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py]: Nit: "in its".

File: tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py (hunk 2 above)
Comment (bebef1987) [inline tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py]: done

File: tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py (hunk 3 above)
Comment (bebef1987) [inline tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py]: done

**Comment by try-server-hook** [comment]
[bebef1987 Florin Strugariu (bebef1987)](https://github.com/bebef1987) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8b6565d34fa7)

**Pull request merged**